spatial-tracking-explainer.md

```diff
@@ -0,0 +1,382 @@
+
+### Stationary Frame of Reference
+A _stationary_ experience is one which does not require the user to move around in space.  This includes several categories of experiences that developers are commonly building today.  "Standing" experiences can be created by passing the `floor-level` subtype.  "Seated" experiences can be created by passing the `eye-level` subtype.  Orientation-only experiences such as 360 photo/video viewers can be created by passing the `position-disabled` subtype.
+
+It is important to note that `XRDevicePose` objects retrieved using the `floor-level` and `eye-level` subtypes may include position information as well as rotation information.  For example, hardware which does not support 6DOF tracking (ex: GearVR) may still use neck-modeling to improve user comfort. Similarly, a user may lean side-to-side on a device with 6DOF tracking (ex: HTC Vive).  It is important for user comfort that developers do not attempt to remove position data from these matrices and instead use the `position-disabled` subtype.  The result is that `floor-level` and `eye-level` experiences should be resilient to position changes despite not being dependent on receiving them.  
+
+#### Floor-level Subtype
+
+The origin of this subtype will be initialized at a position on the floor where it is safe for the user to engage in standing-scale experiences, with a `y` value of `0` at floor level. The exact `x`, `z`, and orientation values will be initialized based on the conventions of the underlying platform for standing-scale experiences. Some platforms may initialize these values to the user's exact position/orientation at the time of creation. Other platforms may place this standing-scale origin at the user's chosen floor-level origin for bounded experiences. It is also worth noting that some XR hardware will be unable to determine the actual floor level and will instead use an emulated floor.
+
+```js
+let xrSession = null;
+let xrFrameOfReference = null;
+
+function onSessionStarted(session) {
+  xrSession = session;
+  xrSession.requestFrameOfReference({ type:'stationary', subtype:'floor-level' })
+  .then((frameOfReference) => {
+    xrFrameOfReference = frameOfReference;
+  })
+  .then(setupWebGLLayer)
+  .then(() => {
+    xrSession.requestAnimationFrame(onDrawFrame);
+  });
+}
+```
+
+#### Eye-level Subtype
```

Comment:
I previously posted a comment about the lack of 'local' FoR and then realized that I haven't finished my reading and removed the comment :) 
So, yeah, the 'eye-level' FoR should work for user-centered UI case I described in issue https://github.com/immersive-web/webxr/issues/389. The only issue I have is the name: I like 'local' better and, according to @casseveritt it is more inlined with OpenXR names.

Reply:
Sounds good!  I'll be filing a separate issue to bikeshed the type names and we can all arm wrestle over it in that thread.